Contents/Code/PAutils.py

```diff
@@ -389,7 +389,7 @@ def postParseTitle(output):
     replace = [('“', '\"'), ('”', '\"'), ('’', '\''), ('W/', 'w/'), ('Aj', 'AJ')]
 
     # Add space after a punctuation if missing
-    output = re.sub(r'(?=[\!|\:|\?|\.](?=(\w{2,}))\b)\S(?!(co\b|net\b|com\b|org\b|porn\b|E\d|xxx\b))', lambda m: m.group(0) + ' ', output, flags=re.IGNORECASE)
+    output = re.sub(r'(?=[\!|\:|\?|\.](?=(\w{2,}))\b)\S(?!(co\b|net\b|com\b|org\b|porn\b|E\d|\b))', lambda m: m.group(0) + ' ', output, flags=re.IGNORECASE)
```

Comment:
needed to match exact id e.g., `https://realjamvr.com/scene/kill-bill-o-ren-ishii-a-xxx-parody/`